packages/vitest-pool-workers/CHANGELOG.md

```diff
@@ -1372,7 +1372,7 @@
   		poolOptions: {
   			workers: {
   				wrangler: {
-  					configPath: "./wrangler.toml",
+  					configPath: "./wrangler.jsonc",
```

Comment:
We shouldn't be modifying the changelog like this. This can (and should) stay as `wrangler.toml`

Reply:
sorry, this was just a search-and-replace unintentional change, thanks a bunch for spotting it, I'll revert it 👍

Reply:
removed 🙂